packages/xrpl/src/models/methods/accountInfo.ts

```diff
@@ -89,6 +145,11 @@ export interface AccountInfoResponse extends BaseResponse {
      * present.
      */
     account_data: AccountRoot & { signer_lists?: SignerList[] }
+
+    /**
+     * A map of account flags parsed out.  This will only be available for rippled nodes 1.11.0 and higher.
+     */
+    account_flags?: AccountInfoAccountFlags
```

Comment:
Is this actually a map of flags, or is the data type an integer? (Do we have the integer mapping?)

Reply:
It is a map of booleans.